classes/local/v1p1/endpoint_test.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
All files should have the Moodle license header, as well as author and Copyright information